sources/packages/backend/apps/workers/src/controllers/application/_tests_/e2e/application.controller-verifyApplicationExceptions.e2e-spec.ts

```diff
@@ -0,0 +1,127 @@
+import { APPLICATION_NOT_FOUND } from "@sims/services/constants";
+import { ApplicationData, ApplicationExceptionStatus } from "@sims/sims-db";
+import {
+  createE2EDataSources,
+  createFakeApplication,
+  E2EDataSources,
+} from "@sims/test-utils";
+import { ICustomHeaders } from "zeebe-node";
+import {
+  createFakeWorkerJob,
+  FAKE_WORKER_JOB_ERROR_CODE_PROPERTY,
+  FAKE_WORKER_JOB_ERROR_MESSAGE_PROPERTY,
+  FAKE_WORKER_JOB_RESULT_PROPERTY,
+  MockedZeebeJobResult,
+} from "../../../../../test/utils/worker-job-mock";
+import { createTestingAppModule } from "../../../../testHelpers";
+import { ApplicationController } from "../../application.controller";
+import {
+  ApplicationExceptionsJobInDTO,
+  ApplicationExceptionsJobOutDTO,
+} from "../../application.dto";
+import { createFakeVerifyApplicationExceptionsPayload } from "./verify-application-exceptions";
+
+describe("ApplicationController(e2e)-verifyApplicationExceptions", () => {
+  let db: E2EDataSources;
+  let applicationController: ApplicationController;
+
+  beforeAll(async () => {
+    const { nestApplication, dataSource } = await createTestingAppModule();
+    await nestApplication.init();
+    db = createE2EDataSources(dataSource);
+    applicationController = nestApplication.get(ApplicationController);
+  });
+
+  it("Should create and associate application exceptions when there are some exceptions for the application.", async () => {
+    // Arrange
+    const fakeApplication = createFakeApplication();
+    fakeApplication.data = {
+      workflowName: "",
+      test: {
+        test: "studentApplicationException",
+      },
+    } as ApplicationData;
+    const savedApplication = await db.application.save(fakeApplication);
+    const verifyApplicationExceptionsPayload =
+      createFakeVerifyApplicationExceptionsPayload(savedApplication.id);
+
+    // Act
+    const result = await applicationController.verifyApplicationExceptions(
+      createFakeWorkerJob<
+        ApplicationExceptionsJobInDTO,
+        ICustomHeaders,
+        ApplicationExceptionsJobOutDTO
+      >(verifyApplicationExceptionsPayload),
+    );
+
+    // Asserts
+    expect(result).toHaveProperty(
+      FAKE_WORKER_JOB_RESULT_PROPERTY,
+      MockedZeebeJobResult.Complete,
+    );
+
+    const updatedApplication = await db.application.findOne({
+      relations: { applicationException: true },
```

Comment:
Can we add the select here also?

Reply:
Not sure if it's my GIT issue. I am seeing the changes

Reply:
I am not able to see any change here also. Was the code changed?

Reply:
I added a "select" for all queries now. Thanks.